eva-lib/src/main/java/uk/ac/ebi/variation/eva/lib/datastore/DBAdaptorConnector.java

```diff
@@ -64,7 +69,36 @@ public static VariantSourceDBAdaptor getVariantSourceDBAdaptor(String species)
         return new VariantSourceMongoDBAdaptor(getCredentials(species, properties),
                 properties.getProperty("eva.mongo.collections.files"));
     }
-    
+
+    public static MongoClient getMongoClient(Properties properties) throws UnknownHostException {
+
+        String authenticationDb = properties.getProperty("eva.mongo.auth.db", null);
+        if (authenticationDb == null || authenticationDb.isEmpty()) {
+            return new MongoClient();
```

Comment:
>without an authentication DB, ... with the hosts and and credentials.

I'm not sure I understand. It's true (I'm fixing this) I can set up a `MongoClient` with the server addresses and without credentials as `new MongoClient(servers, null)`, but I don't think a credentials object can be done without an authentication DB.